.travis.yml

```diff
@@ -3,12 +3,11 @@ language: ruby
 cache: bundler
 
 rvm:
-  # - 1.8.7
-  # - 1.9.2
   - 1.9.3
   - 2.0.0
-  - 2.1
-  - 2.2
+  - 2.1.8
+  - 2.2.4
+  - 2.3.0
```

Comment:
I would like to keep using the short versions (like `2.2`), making sure that the latest minor version available will be picked.